src/datasources/transaction-api/transaction-api.service.spec.ts

```diff
@@ -139,7 +139,7 @@ describe('TransactionApi', () => {
 
       await service.clearLocalBalances(safeAddress);
 
-      expect(mockCacheService.deleteByKey).toBeCalledTimes(1);
+      expect(mockCacheService.deleteByKey).toBeCalledTimes(2);
```

Comment:
`clearLocalBalances` now clears both `Balances` (Transaction Service balances including fiat amounts) and `SimpleBalances` (Transaction Service balances not including fiat amounts).